cypress/e2e/CMS/$BlogSeries/index.js

```diff
@@ -111,6 +124,30 @@ And('blog series description reads {string}', (desc) => {
     cy.get(`p:contains('${desc}')`).should('be.visible');
 });
 
+And('user clicks on {string} sub tab', (contentSubTab) => {
+    cy.get(`ul.admin-list li a:contains("${contentSubTab}")`).click({ force: true });
```

Comment:
this step exist in common - remove it here

Reply:
The step fails with the locator in common . So we are keeping the existing code in index.js for this step.